tests/kernel/init/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  kernel.bootdelay:
+    extra_configs:
+      - CONFIG_BOOT_DELAY=1000
```

Comment:
why we need it, anyway you are adding in prj.conf

Reply:
Removed.